src/items/functions.md

```diff
@@ -123,4 +123,4 @@ As non-Rust calling conventions do not support unwinding, unwinding past the end
 of an extern function will cause the process to abort. In LLVM, this is
 implemented by executing an illegal instruction.
 
-[external blocks]: items/external-blocks.html
```

Comment:
Unrelated to this PR, but why does this keep showing up? And why this time in a file that is otherwise untouched?